### ISSUE TITLE
PET notebooks copy data from wrong place

### ISSUE THREAD

**Issue by KrisThielemans** [SyneRBI/SIRF-Exercises#100]
They copy data from sirf_exercises_path but should be stuff like
```
import sirf.Utilities
data_source_dir = os.path.join(sirf.Utilities.examples_data_path('PET'),'brain')
shutil.rmtree('working_folder/brain',True)
shutil.copytree(data_source_dir ,'working_folder/brain')
```
@ashgillman would you have time for this? Possibly together with #84 but let's not be ambitious!

**Comment by ashgillman** [comment]
This step is already handled in `image_creation_and_simulation.ipynb`. Or have I misunderstood?

```python3
#%% Go to directory with input files
# Adapt this path to your situation (or start everything in the relevant directory)
os.chdir(exercises_data_path('PET'))
# Copy files to a working folder and change directory to where these files are.
# We do this to avoid cluttering your SIRF files. This way, you can delete 
# working_folder and start from scratch.
shutil.rmtree('working_folder/brain',True)
shutil.copytree(os.path.join(examples_data_path('PET'), 'brain'),'working_folder/brain')
os.chdir('working_folder/brain')
```

**Comment by KrisThielemans** [comment]
looks like it. I updated `display_and_projection`. maybe there's some others.

**Comment by ashgillman** [comment]
I understood that the other notebooks aren't necessarily meant to run completely without running `image_creation_and_simulation.ipynb` first (like the brainweb notebooks in Synergistic which the other notebooks depend upon). So does this not hand this?

**Comment by KrisThielemans** [comment]
please check what I did in the `display*` notebook. If you're happy with it, I guess you could copy it (removing most of the comments). If not, modify it first!

**Comment by ashgillman** [comment]
Should be addressed in #101

**Comment by KrisThielemans** [comment]
@ashgillman I believe this was fixed by #101?

**Comment by ashgillman** [comment]
Yes, everything is now run in `DATA/working_folder/...` and data is never copied from examples, etc., but referenced as a full path.
